Company.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+
+namespace task_Dev_3
+{
+     /// <summary>
+    ///  class is designed to store information about employees
+    /// </summary>
+    class Company
+    {
+        Employee junior = new Junior();
+        Employee middle = new Middle();
+        Employee senior = new Senior();
+        Employee lead = new Lead(); 
+
+        /// <summary>
+        /// create two lists for employee storage
+        /// </summary>
+        public List<Employee> employees = new List<Employee>();
+        List<Employee> choosedEmployees = new List<Employee>();
+        int[] numberOfProgrammers;
+
+        public Company()
+        {
+            for (int i = 0; i < 10; i++)
+            {
+                employees.Add(junior);
+                employees.Add(middle);
+                employees.Add(senior);
+                employees.Add(lead);
+            }
+        }
+        /// <summary>
+        /// this method is for adding employees that returns selected employees
+        /// </summary>
+        public List<Employee> GetEmployees(Optimizer opt)
+        {
+            numberOfProgrammers = opt.Choose();
+            for(int i=0; i<numberOfProgrammers[0]; i++)

```

Comment:
numberOfProgrammers[0] 
numberOfProgrammers[1]
numberOfProgrammers[2]

what does it mean? Some magic numbers